test/behaviour/concept/type/thingtype/ThingTypeSteps.java

```diff
@@ -362,4 +400,18 @@ public void thing_type_get_playing_roles_do_not_contain(RootLabel rootLabel, Str
             assertFalse(actuals.contains(roleLabel));
         }
     }
+
+    @Then("{root_label}\\( ?{type_label} ?) get playing roles explicit contain:")
+    public void thing_type_get_playing_roles_explicit_contain(RootLabel rootLabel, String typeLabel, List<Label> roleLabels) {
+        Set<Label> actuals = get_thing_type(rootLabel, typeLabel).asRemote(tx()).getPlaysExplicit().map(Type::getLabel).collect(toSet());
+        assertTrue(actuals.containsAll(roleLabels));
+    }
+
+    @Then("{root_label}\\( ?{type_label} ?) get playing roles explicit do not contain:")
+    public void thing_type_get_playing_roles_explicit_do_not_contain(RootLabel rootLabel, String typeLabel, List<Label> roleLabels) {
+        Set<Label> actuals = get_thing_type(rootLabel, typeLabel).asRemote(tx()).getPlaysExplicit().map(Type::getLabel).collect(toSet());
+        for (Label roleLabel : roleLabels) {
+            assertFalse(actuals.contains(roleLabel));
+        }
```

Comment:
notice that in some cases you've done `for` loop inline and in some places you've expanded with `{`
we should pick one style and try to be consistent (and particularly within one PR)

Reply:
Switched to the braced version everywhere since that seems to be the prevailing style in these tests.